src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -152,7 +157,10 @@ def __call__(self, *wrapped_args, **wrapped_kwargs):
         for name, kwarg in wrapped_kwargs.items():
             kwargs[name] = self._unpack_argument(kwarg, name)
 
-        return self._user_fn(*args, **kwargs)
+        try:
+            return self._user_fn(*args, **kwargs)
+        except SystemError as e:
+            raise exceptions.TaskWrappingError from e
```

Comment:
**thought:** This will catch different op-codes, but are there any other error cases we should think about? We'll see I guess!